crates/swift-bridge-ir/src/codegen/codegen_tests/option_codegen_tests.rs

```diff
@@ -4,6 +4,70 @@ use super::{CodegenTest, ExpectedCHeader, ExpectedRustTokens, ExpectedSwiftCode}
 use proc_macro2::TokenStream;
 use quote::quote;
 
+/// Test code generation for Rust function that accepts and returns an Option<Vec<T>> where T is a
```

Comment:
MINOR:

Can we move this test below `extern_rust_fn_option_primitive`.

I think that the tests are easier to follow if we start with the simple scenarios are gradually build up to more complex scenarios.